start-from-zero/Javascript/src/events/LoginEvents.js

```diff
@@ -0,0 +1,18 @@
+const LoginEvents = function(){
+    this.initEvents = function(build){
+        const form = document.getElementById('login-form');
+        form.addEventListener('submit',function(e){
+            e.preventDefault();
+            const userData = {};
+
+            Array.prototype.forEach.call(form.children,function(item){
+                if(item.name){
+                    userData[item.name] = item.value;
+                }
+            });
+
+            window.dataStorage.set('userData',userData);
+            build(false,true);
```

Comment:
While this is 100% functional it doesn't really tell a programmer much. He has to go to the build definition in order to understand the parameters that you are passing here. Is there any way you can make this more verbose, so that it is easier to understand? Let's talk about this.